src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetTypeFromCLSIDTests.cs

```diff
@@ -16,27 +21,63 @@ public void GetTypeFromCLSID_NoSuchCLSIDExists_ReturnsExpected()
             Assert.NotNull(type);
             Assert.Same(type, Marshal.GetTypeFromCLSID(Guid.Empty));
 
+            Assert.Same(type, Type.GetTypeFromCLSID(Guid.Empty));
+            Assert.Same(type, Type.GetTypeFromCLSID(Guid.Empty, throwOnError: true));
+
             Assert.Throws<COMException>(() => Activator.CreateInstance(type));
         }
 
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
         [PlatformSpecific(TestPlatforms.Windows)]
         public void GetTypeFromCLSID_CLSIDExists_ReturnsExpected()
         {
-            var guid = new Guid("927971f5-0939-11d1-8be1-00c04fd8d503");
-
-            Type type = Marshal.GetTypeFromCLSID(guid);
+            Type type = Marshal.GetTypeFromCLSID(TestCLSID);
             Assert.NotNull(type);
-            Assert.Same(type, Marshal.GetTypeFromCLSID(guid));
+            Assert.Same(type, Marshal.GetTypeFromCLSID(TestCLSID));
+
+            Assert.Same(type, Type.GetTypeFromCLSID(TestCLSID));
```

Comment:
We did not have any test coverage for Type.GetTypeFromCLSID and Type.GetTypeFromProgID